cves/CVE-2017-14056.yml

```diff
@@ -318,14 +336,14 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: There is no problem as long as users are using valid files.
```

Comment:
This is true, but I'm not sure if this is the right interpretation of this principle. From my understanding, secure by default means that default settings/handling are secure and have layers, such that the system should enforce error checking for example. I don't think a system should assume users are inputting valid files.